src/composables/useImageUpload.ts

```diff
@@ -0,0 +1,53 @@
+import { ref } from 'vue';
```

Comment:
The way it's currently used I'd say it doesn't make sense to be composable (consumer gets imageUrl/imageName) from the function call, so refs are there for nothing.

It could just be a function that we call to upload image instead, no vue integration needed.

Reply:
That part makes some sense, but given that we end up using it just as a regular function (+ reactive isUploadingImage) in the makes me think it's over complicated.

We only need to use that logic in single component currently and it will be much faster as non-reactive helper function where isUploadingImage is handled by customer.

When we end up lacking some functionality with that approach we can go back to it and see if reactivity makes sense.